docs/contributing.md

```diff
@@ -41,7 +41,21 @@ $ mkdocs serve
 
 Before a new release, please do the following:
 
-* Add a date to the changelog for this version
-* Run the `DustyRelease` Jenkins job with your new version number
-* Bump the version number in dusty/constants.py to the *next* version
-* Bump the version of the curl command on the Readme, and in `docs/installation.md`
+* Update the changelog
+  * Add a date to the changelog for this version
+  * Add a new version that has an `In progress` title rather than a date
+  * Update the changelog to reflect your new changes with appropriate descriptions
+    * NEW: This change is an addition to the existing code/functionality, and should include a description of what the addition does.
+    * BREAKING: This update changes the existing promises of the dusty CLI API or specs API
+    * FIXED: This is a patch for something broken in the current dusty version
+    * other comments: anything that is not one of the above
```

Comment:
Adding shit to the changelog isn't something you do when you cut a release, it's something you do when you merge new features and fixes and stuff